nstack-translations-generator/Classes/TranslationsGenerator/GeneratorSettings.swift

```diff
@@ -63,8 +65,12 @@ extension GeneratorSettings {
         if let flat = parsedArguments["-flat"] , flat.count == 1 && flat[0] == "1" {
             flatTranslations = true
         }
+        
+        if let _ = parsedArguments["--use-objc"] {
```

Comment:
Just to keep it consistent, it should be one dash instead of two, sorry my fault for misleading you :D